java/com.domain/src/main/test/java/com/domain/http/HttpTest.java

```diff
@@ -0,0 +1,114 @@
+package com.domain.http;
+
+import com.alibaba.fastjson.JSON;
+import com.alibaba.fastjson.JSONObject;
+import com.domain.utils.UUIDUtils;
+import lombok.SneakyThrows;
+import okhttp3.*;
+
+import java.util.concurrent.*;
+
+
+public class HttpTest {
+
+    private static String serverUrl="http://127.0.0.1:8080/";
+
+    private static String accessToken="e3853146-a77e-4f51-b52d-277f719a5849";
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- ee191454-e816-41e6-83fd-21d25e8fe22c -->